armi/physics/executers.py

```diff
@@ -0,0 +1,177 @@
+"""
+Executors are useful for having a standard way to run physics calculations.
+
+They may involve external codes (with inputs/execution/output) or in-memory
+data pathways.
+"""
+
+import os
+
+from armi.utils import directoryChangers
+from armi import runLog
+
+
+class ExecutionOptions:
```

Comment:
Perhaps not now, but something to think about: I think Options stand a better chance of becoming part of various Interface APIs, and therefore may make sense to live in a module of their own.

Reply:
not a bad idea